docs/installation.rst

```diff
@@ -169,16 +183,34 @@ Running the following command in the python (virtual) environment in Terminal/Po
 
     cd OpenQL
     git submodule update --init --recursive
-    python setup.py install
+    pip install -v
```

Comment:
why change the python setup.py to pip install ?

Reply:
Because the former has been part of the issues I've been having with Nikiforos, I think. If you install using `setup.py` directly pip's package management logic is bypassed so pip may or may not be aware of the installation. That's literally the only difference with `pip install`, since `pip` just calls `setup.py`.